### PR TITLE
[Documentation] MQE-1796: Magento Git vs Composer and MFTF

### PR DESCRIPTION
**Pull request by soumyau**
### Fixed Issues (if relevant)

1. magento/magento2-functional-testing-framework#<issue_number>: Issue title
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: docs/guides/git-vs-composer-install.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+###How to download Magento + MFTF via Composer?
+
+Composer install downloads released packages of Magento 2 from the composer repo [https://repo.magento.com](https://repo.magento.com).
+
+All Magento modules and their MFTF tests are put under `<vendor>` directory for convenience of 3rd party developers. With this setup, you can keep your custom modules separate from the core. You can also develop modules in a separate VCS repository and add them to your `composer.json` which will allow them to be installed into the `vendor` directory.
+
+Install guide: [Composer based Installation](https://devdocs.magento.com/guides/v2.3/install-gde/composer.html)
+
+
+###Managing MFTF test artifacts - Composer vs GitHub
```
Comment (jilu1) [inline docs/guides/git-vs-composer-install.md]: To me this section talks more about benefits of using one way or the other. Maybe the title can be more specific for that?

File: docs/guides/git-vs-composer-install.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+#Git vs Composer Installation of Magento with MFTF
+
+
+###How to download Magento + MFTF from GitHub?
```
[the remark this answers is not in the document]
Comment (soumyau) [reply docs/guides/git-vs-composer-install.md]: I changed it to something more generic.

File: docs/guides/git-vs-composer-install.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+If you are planning on contributing a PR to the Magento 2 codebase, you can download Magento 2 from GitHub. Contribution to the codebase is done using the 'fork and pull' model where contributors maintain their own fork of the repo. This repo is then used to submit a pull request to the base repo.
+
+Install guide: [GitHub Installation](https://devdocs.magento.com/mftf/docs/getting-started.html)
+
+###How to download Magento + MFTF via Composer?
```
[the remark this answers is not in the document]
Comment (soumyau) [reply docs/guides/git-vs-composer-install.md]: I changed it to something more generic.

File: docs/guides/git-vs-composer-install.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+#Git vs Composer Installation of Magento with MFTF
+
+
+###How to download Magento + MFTF from GitHub?
+
+If you are planning on contributing a PR to the Magento 2 codebase, you can download Magento 2 from GitHub. Contribution to the codebase is done using the 'fork and pull' model where contributors maintain their own fork of the repo. This repo is then used to submit a pull request to the base repo.
+
+Install guide: [GitHub Installation](https://devdocs.magento.com/mftf/docs/getting-started.html)
```
[the remark this answers is not in the document]
Comment (soumyau) [reply docs/guides/git-vs-composer-install.md]: For github - wanted to point to the steps of installing and using MFTF with git cloning. MFTF comes pre-installed in the composer based setup.

File: docs/guides/git-vs-composer-install.md (hunk 1 above)
Comment (soumyau) [reply docs/guides/git-vs-composer-install.md]: Made this generic to Managing modules - Composer vs Github.

File: docs/guides/git-vs-composer-install.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
+#Git vs Composer Installation of Magento with MFTF
```
Comment (KevinBKozan) [inline docs/guides/git-vs-composer-install.md]: Any place where you use `# Headings` you need to include a space between the `#` and the heading itself

File: docs/guides/git-vs-composer-install.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+####Via composer:
+
+Magento 2 advocates the use of composer for managing modules. When you install a module through composer, it is added to `vendor/<vendor-name>/<module>`
+
+If you are developing your own module or adding MFTF tests to the module, you should not edit `vendor` because a composer update could clobber your changes. Instead, you can override a module under `vendor`, by adding files or cloning your module specific git repo to `app/code/<vendor-name>/<module>`.
+
+If you want to distribute the module and its tests, you can initialize a git repo and create a [composer package](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/package/package_module.html). In this way others will be able to download and install your module and access your tests as a composer package, in their `<vendor>` folder.
+
+
+###MFTF test artifacts location
```
Comment (KevinBKozan) [inline docs/guides/git-vs-composer-install.md]: I don't think that `test artifacts` is the right term, `artifacts` refers to the result of test runs
Use `test materials` instead

File: docs/guides/git-vs-composer-install.md (hunk 5 above)
Comment (soumyau) [reply docs/guides/git-vs-composer-install.md]: fixed

File: docs/guides/git-vs-composer-install.md (hunk 6 above)
Comment (soumyau) [reply docs/guides/git-vs-composer-install.md]: changed them to test materials.

Comment (soumyau) [reply docs/guides/git-vs-composer-install.md]: Oops, two were left out. Fixed them both!

**Pull request merged**